tests/helpers/utils.py

```diff
@@ -51,17 +51,20 @@ def random_mip_1(disable_sepa=True, disable_heur=True, disable_presolve=True, no
 
 
 def random_lp_1():
-    return random_mip_1().relax()
+    model = random_mip_1()
+    model.relax()
+    
+    return model
 
 
 def random_nlp_1():
     model = Model()
 
-    v = model.addVar()
-    w = model.addVar()
-    x = model.addVar()
-    y = model.addVar()
-    z = model.addVar()
+    v = model.addVar(name="v")
+    w = model.addVar(name="w")
+    x = model.addVar(name="x")
+    y = model.addVar(name="y", ub=1.4)
+    z = model.addVar(name="z")
```

Comment:
```suggestion
    z = model.addVar(name="z", ub=4)
```
Then it is actually bounded.